mailbox/api/src/main/java/org/apache/james/mailbox/model/SearchQuery.java

```diff
@@ -633,6 +633,10 @@ public static Criterion flagIsUnSet(Flag flag) {
         return new FlagCriterion(flag, BooleanOperator.unset());
```

Comment:
that means you'll call it with a raw boolean "true" or "false", without meaning, and then you'll have to name it, and then you'll have something very near the actual code, so I prefer to not change it